app/controllers/publishers/vacancies/base_controller.rb

```diff
@@ -26,7 +26,7 @@ def all_steps_valid?
     steps_to_skip = if current_organisation.is_a?(School)
                       []
                     else
-                      vacancy.job_location == "at_central_office" ? [:job_location] : %i[job_location schools]
+                      vacancy.job_location == "central_office" ? [:job_location] : %i[job_location schools]
```

Comment:
you can do
`vacancy.central_office? ? [:job_location] : %i[job_location schools]`